### PR TITLE
Ingester: Conditional put utilizes less than or equal to for records.…

### PR DESCRIPTION
**Pull request by ABPLMC**
… Resolves ingestion queue bug.

### CODE REVIEW

File: ingester/tests/test_storage.py [hunk 1]
```diff
@@ -143,3 +143,58 @@ def test_store_conditional_put_latest_multiple_files(dynamodb_latest_table, dyna
     res = dict(dynamodb_latest_table.get_item(what_where_key=query_what_where))
     assert res['metadata']['start'] == file2['metadata']['start']
 
+
+
+def test_insert_ingestion_issue(dynamodb_latest_table, dynamodb_connection):
```
Comment (bcavagnolo) [inline ingester/tests/test_storage.py:148]: Maybe a better name for this test? Is it something like `test_insert_duplicate_writes_once`?

File: ingester/tests/test_storage.py [hunk 2]
```diff
@@ -143,3 +143,58 @@ def test_store_conditional_put_latest_multiple_files(dynamodb_latest_table, dyna
...
+        },
+        'url': 's3://newfile/url',
+        'create_time': 1500000000000
+    }
+
+    try:
+        storage.store_latest(record1)
+        storage.store_latest(record2)
+    except Exception as e:
+        print(f"Failed to store record: {str(e)}")
```
[the remark this answers is not in the document]
Comment (ABPLMC) [reply ingester/tests/test_storage.py]: Good call, fixed.

File: ingester/tests/test_storage.py (hunk 1 above)
Comment (ABPLMC) [reply ingester/tests/test_storage.py]: Fixed with test_verify_replace_record_same_start. I can change it if we still don't like that.

File: ingester/tests/test_storage.py [hunk 3]
```diff
@@ -127,19 +129,88 @@ def test_store_conditional_put_latest_multiple_files(dynamodb_latest_table, dyna
...
-    records = [dict(i) for i in dynamodb_latest_table.scan()]
+    query_what_where = 'syslog:ground_server2'
+
     res = dict(dynamodb_latest_table.get_item(what_where_key=query_what_where))
     assert res['metadata']['id'] != file1['metadata']['id']
     assert res['metadata']['id'] == file2['metadata']['id']
-    
+
+
+def test_store_conditional_put_same_duplicate_overwrite(dynamodb_latest_table, dynamodb_connection):
```
Comment (alexpaulzor) [inline ingester/tests/test_storage.py:149]: What is the behavior this is testing for? What makes it different from `test_store_conditional_put_latest_multiple_files()`?

Comment (ABPLMC) [reply ingester/tests/test_storage.py:149]: Nothing, it is a duplicate. I'll remove 👍

**Comment by alexpaulzor** [comment]
LGTM! Thank you!

**Pull request merged**